help/configure-who-can-create-channels.md

```diff
@@ -48,5 +48,7 @@ unadvertised web-public channel in a legitimate organization.
 
 ## Related articles
 
-* [Create a channel](/help/create-a-channel)
+* [Channel permissions](/help/channel-permissions)
 * [User roles](/help/user-roles)
```

Comment:
Let's link to /help/user-groups too wherever we link to /help/user-roles just because it's a permission setting.